tests/InterceptorsTest.cc

```diff
@@ -272,8 +273,11 @@ class ConsumerTestInterceptor : public ConsumerInterceptor {
 class ConsumerInterceptorsTest : public ::testing::TestWithParam<std::tuple<TopicType, int>> {
    public:
     void SetUp() override {
-        topic_ = "persistent://public/default/InterceptorsTest-ConsumerInterceptors-" +
-                 std::to_string(time(nullptr));
```

Comment:
You can just splice `GetParam()`.

Reply:
Using `GetParam()` couldn't solve the issue. Multiple tests share the same params. If we want to splice `GetParam()`, we also need to add the `current test method` name to the test topic name. But this PR seems simpler.